sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADJwtGrantedAuthoritiesConverter.java

```diff
@@ -30,13 +34,16 @@
         for (String authority : getAuthorities(jwt)) {
             grantedAuthorities.add(new SimpleGrantedAuthority(authority));
         }
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("User's authorities: {}.", grantedAuthorities);
+        }
         return grantedAuthorities;
     }
 
     private Collection<String> getAuthorities(Jwt jwt) {
         Collection<String> authoritiesList = new ArrayList<String>();
         for (String claimName : WELL_KNOWN_AUTHORITIES_CLAIM_NAMES) {
-            if (jwt.containsClaim(claimName)) {
+            if (jwt.hasClaim(claimName)) {
```

Comment:
nice catch, we should always fix deprecated code, thanks

Reply:
And this method is duplicated with `com.azure.spring.autoconfigure.b2c.AADB2CJwtGrantedAuthoritiesConverter#getAuthorities`

could you also some refactor to remove this duplication code?

Reply:
Yes, I have optimized the converter structure.